ros_speech_recognition/scripts/speech_recognition_node.py

```diff
@@ -288,6 +288,7 @@ def audio_cb(self, _, audio):
             rospy.logdebug("Waiting for result... (Sent %d bytes)" % len(audio.get_raw_data()))
             result = self.recognize(audio)
             confidence = 1.0
+            if result == []: return
```

Comment:
```suggestion
            if len(result) == 0: return
```
this is more common, i think.

Reply:
Thank you!